manifests/init.pp

```diff
@@ -49,10 +50,22 @@
     provider => 'gem',
   }
 
+  if (is_array($scout_roles)) {
+    $scout_roles_config = inline_template('<% @scout_roles.size.to_i %>') ? {
+      0       => fail('Parameter scout_roles can not be an empty array.'),
```

Comment:
**embarrassedface** because I forgot to replace the ````[]```` with ````false```` before I made the PR? :D  I'll fix it